Source/Krypton Components/Krypton.Navigator/View Layout/ViewLayoutInsetOverlap.cs

```diff
@@ -131,18 +131,24 @@ public override void Layout(ViewLayoutContext context)
             Rectangle childRect = ClientRectangle;
 
             // Find the amount of rounding to apply
+            int tempRounding = Convert.ToInt32(Rounding);
```

Comment:
So if I select 1.49
I'm only going to get 2 as the values.
Is that what you have intended, it might look a little odd as the curve will not touch the sides correctly

Reply:
Problem is that `Width`, `X`, `Height` & `Y` are all int values in the `Rectangle` API, unless a custom `Rectangle` class is created